azure-pipelines-templates/device-test.yml

```
@@ -0,0 +1,154 @@
+                      else
+                        find "$UNIT_TEST_RUNSETTINGS_DIR"/TestResults -type f -name "*.trx" -exec rm -f {} \;
+                        ((attempt++))
+                        sleep $DELAY
+                      fi
+                  elif [[ "$RESULT_TEST" =~ "The argument .* is invalid." ]]; then
+                      echo "Test failed as arguments are not valid."
+                      exit 1
+                  else
+                      break
+                  fi
+              done                     
```

yamllint

[error] trailing spaces

(trailing-spaces)